### DISCUSSION TITLE
Comments

### DISCUSSION THREAD

**Discussion by lawrencecchen** [proselog/proselog#82] [Ideas]
Would you be open to a PR that adds threaded comments? Here's one of my projects from last year that implements this:

https://github.com/lawrencecchen/threaded-comments

**Comment by egoist** [comment]
Oh absolutely, I plan to do this myself later but it's better if someone more experienced with this can contribute now 😃

**Comment by egoist** [comment]
However I think this is a low priority, I want other important stuff to be finished first, I will revisit this issue in the future when I think it's time for that.
